storage/shipDB.go

```diff
@@ -151,7 +151,7 @@ var UnknownPos = ShipPos{
 	BowHeading:  uint16(math.NaN()),
 	Course:      float32(math.NaN()),
 	Speed:       float32(math.NaN()),
-	RateOfTurn:  float32(math.NaN()),
+	RateOfTurn:  0,
```

Comment:
0 might also be a valid value, so an obviously invalid value like 999 would be better.

But it turns out I have a ccommit on a local branch that fixes it at the encoding stage by omitting the attribute from the json if NaN.
So  you can just undo this change and I'll push my fix.
Or if you want to I can submit this as-is and then improve upon it with my fix.

Reply:
Perfect, thanks @tormol, I've reverted these in https://github.com/tormol/AIS/pull/6/commits/8af910bd068ea001d0e0000d2387be40a4e0ca38.